src/components/LogIn/LogIn.tsx

```diff
@@ -177,9 +178,9 @@ function UserPassForm() {
     );
     if (bakery) {
       // TODO: Consider displaying an error alert.
-      dispatch(appThunks.connectAndStartPolling()).catch((error) =>
-        console.error(Label.POLLING_ERROR, error),
-      );
+      dispatch(appThunks.connectAndStartPolling())
+        .then(unwrapResult)
```

Comment:
Dispatch will always return a resolved Promise that we need to unwrap as described [here](https://redux-toolkit.js.org/api/createAsyncThunk#handling-thunk-results).